lib/rabbit_feed/consumer_connection.rb

```diff
@@ -42,9 +42,10 @@ def consume &block
       end
 
       sleep # Sleep indefinitely, as the consumer runs in its own thread
-    rescue
+    rescue SystemExit, Interrupt
```

Comment:
[`Raised by exit to initiate the termination of the script.`](http://ruby-doc.org/core-2.2.0/SystemExit.html) - it's not actually called anywhere in the code, I just added it for completeness